testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/BruteForceTest.java

```diff
@@ -384,6 +392,29 @@ public void testNonExistingAccounts() throws Exception {
 
     }
 
+    @Test
+    public void testResetPassword() throws Exception {
+        String userId = adminClient.realm("test").users().search("test-user@localhost", null, null, null, 0, 1).get(0).getId();
+
+        loginSuccess();
+        loginInvalidPassword();
+        loginInvalidPassword();
+        expectTemporarilyDisabled();
+
+        loginPage.resetPassword();
+        resetPasswordPage.assertCurrent();
+        resetPasswordPage.changePassword("test-user@localhost");
+        loginPage.assertCurrent();
+
+        assertEquals("You should receive an email shortly with further instructions.", loginPage.getSuccessMessage());
+
+        events.expectRequiredAction(EventType.SEND_RESET_PASSWORD).user(userId)
```

Comment:
Shouldn't this strictly be a SEND_RESET_PASSWORD_ERROR event?

Reply:
@stianst I don't think so. If the expected outcome is:

`User would receive a success message (that an email has been sent) even though no password reset email is sent due to the user being temporarily disabled.`

Then the event should be a successful event, not an error.

Reply:
The email wasn't actually sent though. The error was just masked from the user, but shouldn't be from the admin.